syne_tune/optimizer/baselines.py

```diff
@@ -733,6 +738,142 @@ def __init__(
+    """
+    Uses :class:`~syne_tune.optimizer.schedulers.multiobjective.MultiObjectiveMultiSurrogateSearcher`
+    with one standard GP surrogate model per metric (same as in
+    :class:`BayesianOptimization`, together with the
+    :class:`~syne_tune.optimizer.schedulers.multiobjective.MultiObjectiveLCBRandomLinearScalarization`
+    acquisition function.
+
+    If `estimators` is given, surrogate models are taken from there, and the
+    default is used otherwise. This is useful if you have a good low-variance
+    model for one of the objectives.
+
+    :param config_space: Configuration space for evaluation function
+    :param metric: Name of metrics to optimize
+    :param mode: Modes of optimization. Defaults to "min" for all
+    :param random_seed: Random seed, optional
+    :param estimators: Use these surrogate models instead of the default GP
+        one. Optional
+    :param kwargs: Additional arguments to
+        :class:`~syne_tune.optimizer.schedulers.FIFOScheduler`. Here,
+        ``kwargs["search_options"]`` is used to create the searcher and its
+        GP surrogate models.
+    """
+
+    def __init__(
+        self,
+        config_space: Dict[str, Any],
+        metric: List[str],
+        mode: Union[List[str], str] = "min",
+        random_seed: Optional[int] = None,
+        estimators: Optional[Dict[str, Estimator]] = None,
+        **kwargs,
+    ):
+        searcher_kwargs = _create_searcher_kwargs(
+            config_space, metric, random_seed, kwargs
+        )
+        searcher_kwargs["mode"] = mode
+        if estimators is None:
+            estimators = dict()
+        else:
+            estimators = estimators.copy()
+        if isinstance(mode, str):
+            mode = [mode] * len(metric)
+        if "search_options" in kwargs:
+            search_options = kwargs["search_options"].copy()
+        else:
+            search_options = dict()
+        search_options["no_fantasizing"] = True
+        for _metric, _mode in zip(metric, mode):
+            if _metric not in estimators:
+                estimators[_metric] = create_gaussian_process_estimator(
+                    config_space=config_space,
+                    metric=_metric,
+                    mode=_mode,
+                    search_options=search_options,
+                )
+        searcher = MultiObjectiveMultiSurrogateSearcher(
+            config_space=config_space,
+            metric=metric,
+            mode=mode,
+            estimators=estimators,
+            scoring_class=partial(
+                MultiObjectiveLCBRandomLinearScalarization, random_seed=random_seed
+            ),
+            random_seed=random_seed,
+        )
+        super().__init__(
+            config_space=config_space,
+            metric=metric,
+            mode=mode,
+            searcher=searcher,
+            random_seed=random_seed,
+            **kwargs,
+        )
+
+
+class NSGA2(FIFOScheduler):
```

Comment:
Has anything changed here? Because this wrapper is already in` baselines.py`

Reply:
Do you mean `NSGA2`? Yes, that is already there. I think diff is messing things up.